### DISCUSSION TITLE
Dashboard List panel - add option to show the tags of each dashboard in the list

### DISCUSSION THREAD

**Discussion by hagaym** [grafana/grafana#37319] [Dashboards, Panels & Visualizations]
Just like the list of dashboards is presented in a Folder.
This way it will be possible to have several 'home' dashboard, for example for each dev-team.

In this context, it will also be nice to have a search for this panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
